### PR TITLE
Remove reliance on user's account balance

### PR DESCRIPTION
**Pull request by triceo**
When Zonky moves to implement PSD2, we will lose access to all financial information, incl. user's account balance. This change removes all reliance on it from the robot.

### CODE REVIEW

File: robozonky-strategy-natural/src/main/antlr4/imports/Tokens.g4 [hunk 1]
```diff
@@ -19,11 +19,6 @@ targetPortfolioSizeExpression returns [long result] :
     {$result = $maximumInvestmentInCzk.result;}
 ;
 
-targetBalanceExpression returns [long result] :
```
Comment (triceo) [inline robozonky-strategy-natural/src/main/antlr4/imports/Tokens.g4]: FYI @jhrcek This is going away in 5.4.0, on account of Zonky implementing the PSD2 regulation. Also, it is not the only change in 5.4 - PSD2 will force us to make many more changes.

**Comment by triceo** [comment]
Sonar failed due to some weird JVM SIGSEGV. Can reproduce locally. Will have to be investigated independently.

File: robozonky-strategy-natural/src/main/antlr4/imports/Tokens.g4 (hunk 1 above)
Comment (jhrcek) [reply robozonky-strategy-natural/src/main/antlr4/imports/Tokens.g4]: Do you have more details about the planned changes? Can you share them or are they available somewhere? PM me on social media if you can share more :-)

Comment (triceo) [reply robozonky-strategy-natural/src/main/antlr4/imports/Tokens.g4]: Sending PM.

**Pull request merged**